packages/neuron-ui/src/components/Receive/index.tsx

```diff
@@ -10,33 +10,79 @@ declare global {
   }
 }
 
-const QRCodePanel = styled.div`
-  margin-top: 30px;
-  text-align: center;
+const AddressTip = styled.div`
+  color: black;
+  font-size: 18px;
+  font-weight: bold;
+  margin: 15px 0 0 30px;
+`
+
+const AddressPanel = styled.div`
+  dispaly: flex;
+  display: -webkit-flex; /* Safari */
+  flex-direction: row;
+  margin: 10px 0 0 30px;
 `
 
 const Address = styled.div`
   color: black;
-  margin-top: 50px;
+  margin-right: 10px;
+`
+
+const CopyImage = styled.img`
+  width: 15px;
+  height: 20px;
+  padding-top: 5px;
+`
+
+const QRCodePanel = styled.div`
+  margin: 50px 0 0 30px;
+`
+
+const QRCodeModal = styled.div`
   text-align: center;
 `
 
-const Receive = () => {
+const Receive = ({ address }: { address: string }) => {
   const [t] = useTranslation()
-  const address: string = '0x0da2fe99fe549e082d4ed483c2e968a89ea8d11aabf5d79e5cbf06522de6e674'
+  const [show, setShow] = useState(false)
```

Comment:
`show` is too generic. Perhaps something like `showLargeQR`?

Reply:
OK